### DISCUSSION TITLE
Jetbrains: Getting suggestions in the python console but unable to accept them.

### DISCUSSION THREAD

**Discussion by dmwyatt** [community/community#6653] [Copilot Conversations]
The Jetbrains CoPilot plugin is giving buggy suggestions when adding code in the python console.

![image](https://user-images.githubusercontent.com/163359/138564258-2dc7993e-1bb2-4cad-b8b9-0d562fac83cb.png)

Note the extra `d` characters.  Also after I delete that code, the suggestion continues to stick around:

![image](https://user-images.githubusercontent.com/163359/138564281-1b04c904-24ba-4599-ab29-cc98ae9821dc.png)

Pressing Tab or Enter doesn't seem to accept the suggestion or basically do anything that you'd expect.

I recommend either preventing suggestions in the REPL or properly implementing them.

**Comment by hsenag** [comment]
Thanks a lot for reporting this - we'll take a look.

**Reply by jansorg** [reply]
@dmwyatt The next update will turn off the completions in the REPL editor, although we couldn't reproduce the problem with the extra `d` character.
If you edit code in a regular editor of your IDE, do the completions appear okay or are there also extra character shown?

**Reply by dmwyatt** [reply]
Completions seem to work fine in the regular editor.

**Reply by Jivko1212** [reply]
Is it possible that an option is added to use copilot in Pycharm console. I do most of the prototyping there and cannot use copilot, because it only works in the regular editor. It would save a lot of typing if it was available there.
